help/en/html/hardware/loconet/Uhlenbrock.shtml

```diff
@@ -57,7 +57,9 @@
       <h2>Limitations</h2>
 
       <ul>
-        <li>Full support of Intellibox-II has been added to JMRI in
+        <li><span class="since">Since
+        <a href="https://jmri.org/releasenotes/jmri3.11.1.shtml" target="_blank">
+        JMRI 4.11.1</a></span>Full support of Intellibox-II has been added to JMRI in
         version 3.11.1 and up. It is not available in previous
         versions 3.10 and below.</li>
```

Comment:
Sorry for missing this before. But the link points to `jmri3.11.1.shtml` but the text in the link is `JMRI 4.11.1`.